src/sockperf.cpp

```diff
@@ -1734,7 +1733,11 @@ static int resolve_sockaddr(const char *host, const char *port, int sock_type,
     if (host != NULL) {
         std::string path = host;
         struct sockaddr_store_t *tmp = ((sockaddr_store_t *)addr);
+#ifdef __linux__
         if (path.size() > 0 && path[0] == '/') {
+#elif __windows__
+        if (path.size() > 0) {
```

Comment:
You broke support for IPv4 and IPv6 on Windows by this change!

Reply:
please explain, not sure how. 
This code is for AF_UNIX so I am not understand how it's related to IPV4 IPV6

Reply:
Could you please show this socket in the output of `netstat -na`? Probably you're using an AF_UNIX socket with name "127.0.0.1" instead of TCP socket with address 127.0.01.

Reply:
OK, yes you absolutely right.! 
So I'll revert to previous code, but then we force the user to use `\` for windows, and it prevents him from using other directory, e.g `D:` 
to reach `D:` he can't use `\D:` it's not legal

Reply:
1. You can add some heuristics (for example, an address that _contains_ `\` or `/` can not be an IPv4 or IPv6 address).
2. You also can force the user to use UNC path (starting by `\\`). Please refer to [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats#unc-paths) about UNC paths and DOS device paths.

Anyway we need to describe in README how to use DOS or UNC-style paths in command line and feedfile.

Reply:
I tried with `\\` it fails 

`WinSockperf.exe sr --addr \\C:\tmp\test --tcp`  - Fail 
`WinSockperf.exe sr --addr \C:\tmp\test --tcp`  - Fail 
`WinSockperf.exe sr --addr \.\tmp\test --tcp` - Success (take relative directory)
`WinSockperf.exe sr --addr \tmp\test --tcp` - Success (starts from C: directory)

Reply:
Please read "DOS device paths" chapter carefully.

Reply:
Sometimes it starts with `\` sometimes not.. 

![path windows](https://user-images.githubusercontent.com/44922030/184103547-c83fe811-1c02-4cad-b930-a7c352b92765.JPG)